test/common/test-functions.js

```diff
@@ -259,3 +259,37 @@ function ph_wait_cond(cond, timeout) {
         step();
     });
 }
+
+function currentFrameAbsolutePosition() {
+  let currentWindow = window;
+  let currentParentWindow;
+  let positions = [];
+  let rect;
+
+  while (currentWindow !== window.top) {
+    currentParentWindow = currentWindow.parent;
```

Comment:
Would be nice to have a consistent 4-space indentation here.

Reply:
Yes, good point.  (This is obviously stackoverflowed...)